### PR TITLE
Make examples consistent and document the likely non-RGB types

### LINKED ISSUE #878: Examples are inconsistent in terms of color order for WS2812(B) and NeoPixel
The color signalling order for NeoPixels (the WS2812 variety), WS2812, WS2812B, WS2852, and SK6812 LEDs are all `GRB`, not `RGB`. However, the examples are unclear on this matter.

For instance, [Blink.ino](https://github.com/FastLED/FastLED/blob/master/examples/Blink/Blink.ino) shows many pixel types and their configurations, but of the types above only the default NeoPixel one will work right "out of the box" - the WS2812, WS2812B, WS2852, and SK6812 (where present) will not, as they specify the `RGB` color order, not their normal `GRB` order.

It would be good to have a small comment by the NeoPixel line noting its implicit default to `GRB` signalling, and the WS2812/WS2812B/WS2852/SK6812 ones (where present) should actually specify `GRB` rather than `RGB`.

I've seen notes suggesting inconsistencies within these groups, but I suspect this is down to users mixing up APA102 / SK6822 with WS2812 variants above (APA102 seems to have several color orders - but I've yet to see a properly marked WS2812 vary from GRB - if one does, it's not the norm).

Making the examples more explicit about NeoPixel / WS2812 / WS2812B / WS2852 / SK6812 being typically `GRB` will help reduce user confusion.

### PR DESCRIPTION
**Pull request by MartyMacGyver**
Resolves #878

### CODE REVIEW

File: examples/RGBCalibrate/RGBCalibrate.ino [hunk 1]
```diff
@@ -65,8 +88,7 @@ void loop() {
     leds[2] = CRGB(0,255,0);
     leds[3] = CRGB(0,0,255);
     leds[4] = CRGB(0,0,255);
-    leds[5] = CRGB(0,0,255);
-    leds[6] = CRGB(0,0,0);
```
Comment (MartyMacGyver) [inline examples/RGBCalibrate/RGBCalibrate.ino]: Noticed and fixes a latent defect here - the array is only 6 entries long, so this is a buffer overflow as-is.

**Comment by skypanther** [comment]
Looks like this also resolves #877

File: examples/RGBCalibrate/RGBCalibrate.ino (hunk 1 above)
Comment (marmilicious) [reply examples/RGBCalibrate/RGBCalibrate.ino]: leds[5] still needs to be set to blue (0,0,255) though.

Comment (MartyMacGyver) [reply examples/RGBCalibrate/RGBCalibrate.ino]: I thought the last led should still be black as apparently was intended in the original code, but I can easily change this.

Comment (MartyMacGyver) [reply examples/RGBCalibrate/RGBCalibrate.ino]: I've made the change

**Comment by MartyMacGyver** [comment]
This covers #928 as well.

**Comment by kriegsman** [comment]
Thank you for the contributions and the help!

**Comment by MartyMacGyver** [comment]
Thanks for accepting my changes! Glad to help out.

**Pull request merged**